### PR TITLE
Creating About page

### LINKED ISSUE #34: Adding a new file
- Adding a new HTML file so we can add some data into the page in the future

### PR DESCRIPTION
**Pull request by Leeadar**
- Creating a new about.html file in the templates directory
- Adding about function to views.py
- Adding a new path in urls.py to the page

After running 'vagrant up' the 'about' page can be accessed at:
> http://127.0.0.1:8000/about/

closes #34

### CODE REVIEW

**Review by nadavspinco — COMMENTED** [review]
I think that you should open a different folder for the about.html file, or change the directory name to one that represents HTML files.
Even the apphomepage name I chose seems wrong right now because of a path :
apphomepage/templates/apphomepage/...

File: apphomepage/templates/apphomepage/about.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (Yarboa) [inline apphomepage/templates/apphomepage/about.html:1]: I have just notice that settings include application called apphome page
Is it really the name of the application?

Will you have more templates? for this application? consider giving different app name where we could add more templates under this one.

Comment (Yarboa) [reply apphomepage/templates/apphomepage/about.html:1]: I also see anothe PR without reviewers 
https://github.com/beyond-io/picATrip/issues/35 it is dealing with the above

Comment (Leeadar) [reply apphomepage/templates/apphomepage/about.html:1]: Do you suggest to change the name of the application in this PR?
Or create a new PR that will deal with #35 ?

**Comment by Leeadar** [comment]
I created a new commit  and changed the name of the directory from 'apphomepage' to 'pickATrip_django_apps'

**Review by nadavspinco — APPROVED** [review]
I cloned the repository and everything works as expected!

**Comment by guy9050** [comment]
Can you make the [first commit](https://github.com/beyond-io/picATrip/pull/33/commits/08f01131eb18bd1249474aab700afdcedbc28c01) more descriptive? something like "rename directory X to Y"

File: apphomepage/templates/apphomepage/about.html (hunk 1 above)
Comment (guy9050) [reply apphomepage/templates/apphomepage/about.html]: @Yarboa okay now? (if yes please resolve)

**Comment by Leeadar** [comment]
@guy9050 Thank you for your review, I made the change.

**Review by guy9050 — APPROVED** [review]
lgtm

**Pull request merged**